azure/eventhub_log_forwarder/event_hub.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "namespaceName": {
+      "type": "string",
+      "defaultValue": "datadog-eventhub-namespace",
+      "metadata": {
+        "description": "Name of EventHub namespace"
+      }
+    },
+    "eventhubSku": {
+      "type": "string",
+      "allowedValues": [ "Basic", "Standard" ],
+      "defaultValue": "Standard",
+      "metadata": {
+        "description": "Specifies the messaging tier for the EventHub Namespace."
+      }
+    },
+    "skuCapacity": {
+      "type": "int",
+      "allowedValues": [ 1, 2, 4 ],
+      "defaultValue": 1,
+      "metadata": {
+        "description": "MessagingUnits for premium namespace"
+      }
+    },
+    "eventHubName": {
+      "type": "string",
+      "defaultValue": "datadog-eventhub",
+      "metadata": {
+        "description": "Name of Event Hub"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2018-01-01-preview",
```

Comment:
Same q about the preview version

Reply:
Yeah that one also appears to be the latest: https://docs.microsoft.com/en-us/azure/templates/microsoft.eventhub/namespaces/eventhubs